webapp/src/Service/ConfigurationService.php

```diff
@@ -183,6 +185,150 @@ function (ConfigCacheInterface $cache) {
         return $specification;
     }
 
+    /**
+     * Save the changes from the given request
+     *
+     * @param array $data
+     * @param EventLogService $eventLog
+     * @param DOMJudgeService $dj
+     *
+     * @throws NonUniqueResultException
+     */
+    public function saveChanges(
+        array $dataToSet,
```

Comment:
Nitpick: the variable name mismatches `$data` above.

Reply:
Whoops had `$data` before but that variable was already used.